conans/test/unittests/model/test_conf.py

```diff
@@ -9,8 +9,9 @@
 @pytest.fixture()
 def conf_definition():
     text = textwrap.dedent("""\
-        tools.microsoft.msbuild:verbosity=minimal
-        user.company.toolchain:flags=someflags""")
+    tools.microsoft.msbuild:verbosity=minimal
```

Comment:
The previous indent wasn't bad, it was actually the one closer to what "black" formatter is doing (which we might probably adopt at some point). In any case it is a change that doesn't help in the review, as it seems initially you are changing tests (more than necessary)